packages/generator/src/downloadRamlsGulpTaskHelpers.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from "path";
+import _ from "lodash";
+
+import { diffRaml, RamlDiff } from "@commerce-apps/raml-toolkit";
+import { generatorLogger } from "./logger";
+
+/**
+ * Extracts all the RAML file names from the specified config file.
+ *
+ * @param configPath - Target config file
+ *
+ * @returns An array of `assetId/fileName` for every api in the input file
+ */
+export function listRamlsFromConfig(configPath: string): string[] {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const config = require(configPath);
+  const ramls = [];
+  for (const apiFamily of Object.keys(config)) {
+    for (const api of config[apiFamily]) {
+      ramls.push(path.join(api.assetId, api.fatRaml.mainFile));
+    }
+  }
+
+  return ramls;
+}
+
+/**
+ * Compares the arrays and returns all the common and exclusive elements.
```

Comment:
```suggestion
 * Compares two arrays and returns their intersection and symmetric difference.
```
Fancy math terms!

Reply:
Sounds too technical but happy to change if it makes you happy 😁